addons/account/models/chart_template.py

```diff
@@ -1080,6 +1103,40 @@ def get_repartition_line_create_vals(self, company):
             }))
         return rslt
 
+    def _get_repartition_line_create_vals_complete(self, company):
+        """
+        This function returns a list of values to create the repartition lines of a tax based on
+        one or several account.tax.repartition.line.template. It mimicks the function get_repartition_line_create_vals
+        but adds the missing field account_id (account.account)
+
+        Returns a list of (0,0, x) ORM commands to create the repartition lines starting with a (5,0,0)
+        command to clear the repartition.
+        """
+        rslt = self.get_repartition_line_create_vals(company)
+        for idx, template_line in zip(range(1, len(rslt)), self):  # ignore first ORM command ( (5, 0, 0) )
```

Comment:
@JulienVR just so you know: you could have use the `enumerate` built-in instead to avoid those `zip(range(len(` shenanigans.

It is negligible in terms of perf considering how slow the whole system is but it makes the code much more readable. (and it is still some footprints that can be easily claimed)